apps/emqx/test/emqx_cm_SUITE.erl

```diff
@@ -353,10 +353,12 @@ test_stepdown_session(Action, Reason) ->
 %% to sync with the pool workers.
 %% The number of tasks should be large enough to ensure all workers have
 %% the chance to work on at least one of the tasks.
-flush_emqx_pool() ->
+flush_emqx_cm_pool() ->
```

Comment:
Nit: this seems almost identical to `emqx_pool:flush_async_tasks/1`?

Reply:
Indeed, replaced it with `emqx_pool:flush_async_tasks/1` and removed, thanks!

Reply:
Sorry, failed to push this fix :see_no_evil: , will issue a follow-up PR..